README.md

```diff
@@ -22,13 +22,15 @@ Manifests:
             - "bar"
           # Explicit config for "other" IoT Things
           shadowDocuments:
-          - thingName: "MyThing"
-            classic: false
-            namedShadows:
-            - "foo"
-            - "bar"
-          - thingName: "OtherThing"
-          - thingName: "YetAnotherThing"
+            <thingName>:
+              classic: false
+              namedShadows:
+              - "foo"
+              - "bar"
+            <thingName>:
+              classic: true
+            <thingName>:
+              classic: false
```

Comment:
I'm confused what this configuration will actually do (don't sync classic shadows and there are no named shadows).

Reply:
This basically means only sync the classic shadow for the thing and no named shadows.

Reply:
```
    <thingName>:
              classic: false
```
This means don't sync the classic shadow. There are no named shadows to sync either - so it won't actually sync anything

Reply:
```
    <thingName>:
              classic: false
```

This means don't sync the classic shadow. There are no named shadows to sync either - so it won't actually sync anything